packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -578,33 +578,30 @@ class IOSDeviceLogReader extends DeviceLogReader {
   String get name => device.name;
 
   @override
-  List<VMService> get connectedVMServices => _connectedVMServices;
-  List<VMService> _connectedVMServices;
+  VMService get connectedVMService => _connectedVMService;
+  VMService _connectedVMService;
 
   @override
-  set connectedVMServices(List<VMService> connectedVMServices) {
+  set connectedVMService(VMService connectedVMServices) {
```

Comment:
nit. the parameter should be `connectedVmService`

Reply:
Fixed!